sbirez/management/commands/importtopics.py

```diff
@@ -27,15 +27,15 @@ def handle(self, mdb_filename, solicitation_name, clear=False, **options):
             raise OSError("MS Access database %s not found" % mdb_filename)
 
         # Confirm that solicitation_name is for solicitation in the database
-        proc = subprocess.Popen(#'ls',
```

Comment:
That’d be preferable. This is a hacky way to do it, and I just made it hackier. :)
> On Sep 21, 2015, at 9:06 AM, Catherine Devlin <notifications@github.com> wrote:
> 
> In sbirez/management/commands/importtopics.py <https://github.com/18F/afsbirez/pull/208#discussion_r39967533>:
> 
> > @@ -27,15 +27,15 @@ def handle(self, mdb_filename, solicitation_name, clear=False, **options):
> >              raise OSError("MS Access database %s not found" % mdb_filename)
> >  
> >          # Confirm that solicitation_name is for solicitation in the database
> > -        proc = subprocess.Popen(#'ls',
> Thanks! Really, now that I think of it, it should be possible (and would be cleaner) to go through Django's ORM... maybe I should do that today. Maybe when I look I'll remember why I didn't do that in the first place.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/18F/afsbirez/pull/208/files#r39967533>.
>